pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeature/acmfeature/dependencies.yaml

```diff
@@ -17,19 +17,21 @@ kind: Project
 metadata:
   annotations:
     cnrm.cloud.google.com/auto-create-network: "false"
+    cnrm.cloud.google.com/deletion-policy: "abandon"
   name: acm-${uniqueId}
 spec:
-  name: acm-${uniqueId}
-  organizationRef:
-    external: ${TEST_ORG_ID}
+  resourceID: ${TEST_DEPENDENT_NO_NETWORK_PROJECT_ID}
+  name: "Dependent Project"
+  folderRef:
+    external: ${TEST_FOLDER_ID}
   billingAccountRef:
     external: ${TEST_BILLING_ACCOUNT_ID}
```

Comment:
Just to double check: is it the billing account ID we've used for pre creation?

Reply:
Yes. 
Due to the GCP billing account restriction, we cannot associate dynamically created test projects to this billing account. Thus we introduced the "dependent test project", which pre-created by cnrm-eap-prow@cnrm-eap.iam.gserviceaccount.com